pkg/controller/logstorage/secrets/secret_controller.go

```diff
@@ -261,9 +261,6 @@ func (r *SecretSubController) Reconcile(ctx context.Context, request reconcile.R
 	// Provision secrets and the trusted bundle into the cluster.
 	hdler := utils.NewComponentHandler(reqLogger, r.client, r.scheme, ls)
 
-	// Determine if Kibana should be enabled for this cluster.
-	kibanaEnabled := !operatorv1.IsFIPSModeEnabled(install.FIPSMode) && !r.multiTenant
```

Comment:
Same here I'd like to keep `kibanaEnabled := !r.multiTenant` for documentation purposes.

Reply:
I addressed this in the code using comments only.